src/components/transactions/transactionsComponent.js

```diff
@@ -39,7 +39,7 @@ class Transactions extends React.Component {
           length: parseInt(res.count, 10),
         });
       })
-      .catch(error => console.error(error.message));
+      .catch(error => console.error(error.message)); //eslint-disable-line
```

Comment:
I see that finally somebody got annoyed enough by the warning in console :-) Though I would prefer to disable the specific rule. In this case a believe it is `//eslint-disable-line no-console`